packages/nameguard-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@namehash/nameguard",
-  "version": "0.0.18",
+  "version": "1.0.0",
```

Comment:
Hmm. I was hoping to make the v1.0 release more symbolic once we reach a special milestone of maturity. What do you think? Appreciate your advice. If there's a practical reason to update this to v1.0 now then happy to proceed.

Reply:
Fixed